plugins/log.js

```diff
@@ -1,39 +1,19 @@
 const fs = require("fs");
-const logFile = fs.createWriteStream("screeps.log", { flags: "a" });
-const errorLogFile = fs.createWriteStream("screeps.errors.log", { flags: "a" });
 
 module.exports = function(multimeter) {
-  let enabled = multimeter.config.log === "true";
+  let log = multimeter.config.logFilename;
+  if (!log) return;
+  let errorLog = log || multimeter.config.errorLogFilename;
```

Comment:
Similarly, this actually prevents you from *disabling* error logs if you want (which, I mean, why would you, but still). A better approach might be:

```javascript
let errorLogFilename =
  typeof multimeter.config.errorLogFilename === 'string'
    ? multimeter.config.errorLogFilename
    : null;

const logFile = log && fs.createWriteStream(...)
const errorLogFile = errorLogFilename && fs.createWriteStream(...)
```

And then of course making sure that the `logFile` and `errorLogFile` variables aren't null before using them.